e2e/docker/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM envoyproxy/envoy:v1.18.3 AS envoy
+WORKDIR /app
+RUN cp $(which envoy) ./envoy
+
+FROM golang:1.16 AS go
+
+FROM tinygo/tinygo:0.18.0
+ENV PATH $PATH:/usr/local/go/bin
+RUN apt-get clean && \
+    apt-get update && \
+    apt-get install -y make
+COPY --from=envoy /app/envoy /usr/local/bin
+COPY --from=go /usr/local/go /usr/local/go
```

Comment:
I think this could break the assumption that Go's binary version matches the version of std libs.

Reply:
So I suggest that Installing TinyGo in go:1.16 container following https://tinygo.org/getting-started/install/linux/#ubuntudebian